.changesets/config_simon_router_version_in_cache_key.md

```diff
@@ -0,0 +1,7 @@
+### Distributed query plan cache keys include the Router version number ([PR #6406](https://github.com/apollographql/router/pull/6406))
```

Comment:
I put this (though filename prefix) in the "Configuration" section of the changelog, but none of the options in `xtask changeset create` seemed like a great fit. 

`CHANGELOG.md` has some `> [!IMPORTANT]` sections, is that done manually when editing for a release?

Reply:
we do that manually in the release PR, since it has to go in the top of the changelog